src/database/migrations/20230401104008-create-products.js

```diff
@@ -37,10 +37,10 @@ module.exports = {
       collectionId: {
         type: Sequelize.UUID,
         allowNull: false,
+        onDelete: 'CASCADE',
         references: {
           model: 'collections',
           key: 'id',
-          onDelete: 'CASCADE',
         },
```

Comment:
Please don't edit migration, if you want to alter the table you can create new migration

Reply:
Yes, Created another Migration instead.